src/model/guild/role.rs

```diff
@@ -10,10 +10,10 @@ use crate::model::utils::is_false;
 
 /// Information about a role within a guild.
 ///
-/// A role represents a set of permissions, and can be attached to one or multiple users. A role has
-/// various miscellaneous configurations, such as being assigned a colour. Roles are unique per
-/// guild and do not cross over to other guilds in any way, and can have channel-specific permission
-/// overrides in addition to guild-level permissions.
+/// A role represents a set of permissions, and can be attached to one or multiple users. A role
+/// has various miscellaneous configurations, such as being assigned a colour. Roles are unique per
+/// guild and do not cross over to other guilds in any way, and can have channel-specific
+/// permission overrides in addition to guild-level permissions.
```

Comment:
Doesn't rustfmt deal with this? This also doesn't make any difference to the userfacing docs, so I don't see a point?

Reply:
Yeah I guess you're right - the `/// ` at the start of the line isn't counted by rustfmt for the line length, so that's why vim still wraps the line. Should I undo this change?

Reply:
It's fine if your editor is wrapping it anyway, because it would just cause pain to keep it as-is for you.